realm/realm-library/src/androidTest/java/io/realm/RealmResultsTests.java

```diff
@@ -1311,6 +1311,21 @@ public void distinct() {
     }
 
     @Test
+    public void distinct_resultsWithTableView() {
```

Comment:
The name of this unit test could be clearer IMO. `distinct_restrictedByPreviousDistinct` or something like that would be clearer.